desktop/apps/auction2/components/filter_sort/index.js

```diff
@@ -10,6 +10,10 @@ function FilterSort(props) {
     updateSortAction
   } = props
   const selectedSort = filterParams.sort
+  const itemHeight = 37
```

Comment:
It's the height of an individual select option. It's probably better to calculate the dynamically, but I wasn't sure if it was odd to stick `$` in this react component? Happy to change that if you think it's better.